MasterPassword/C/mpw-util.c

```diff
@@ -145,6 +145,39 @@ static int putvar(int c) {
 }
 #endif
 
+
+void mpw_toggle_term_echo () {
+   static sigset_t sigOld;
+   static struct termios termOld;
+   static FILE *fp = NULL;
+
+   if (!fp) {
+       if ((fp = fopen( ctermid(NULL), "r+")) == NULL)
+            return;
+
+        sigset_t sig;
+        struct  termios term;
+
+        setbuf( fp, NULL);
+        sigemptyset( &sig );
+        sigaddset( &sig, SIGINT); /* block SIGINT */
+        sigaddset( &sig, SIGTSTP); /* block SIGTSTP */
+        sigprocmask( SIG_BLOCK, &sig, &sigOld);
```

Comment:
It is optional, it is only for robustness. I use it for another function type `getpass`.
I will remove.